scripts/gen_kubeconfig.sh

```diff
@@ -13,6 +13,16 @@
 # limitations under the License.
 ######################################################################
 
+# Determine the command to use.
+if [ -x "$(command -v kubectl)" ]; then
+  cmd='kubectl'
+elif [ -x "$(command -v oc)" ]; then
+  cmd='oc'
+else
+  echo "Neither kubectl nor oc are not installed. Verify setup."
```

Comment:
Could you remove 'not' from this sentence? 
Neither kubectl nor oc **is** installed.